### PR TITLE
Repair Docker Build

### PR DESCRIPTION
**Pull request by willronchetti**
- Updates GPG key configuration/import for nginx so `docker build` can complete

### CODE REVIEW

File: deploy/docker/production/install_nginx.sh [hunk 1]
```diff
@@ -8,18 +8,17 @@ export PKG_RELEASE=1~buster
 # Securely provisions deps, installs Nginx, then removes all artifacts
 set -x \
     && apt-get update \
-    && apt-get install --no-install-recommends --no-install-suggests -y gnupg1 ca-certificates \
-    && \
-    NGINX_GPGKEY=573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62; \
+    && apt-get install --no-install-recommends --no-install-suggests -y gnupg gnupg1 ca-certificates \
+    && gpg --refresh-keys && \
+    NGINX_GPGKEY=ABF5BD827BD9BF62; \
```
Comment (netsettler) [inline deploy/docker/production/install_nginx.sh:13]: That's a lot fewer digits. Is it right?

File: Dockerfile [hunk 2]
```diff
@@ -48,8 +48,10 @@ RUN apt-get update && apt-get upgrade -y && \
     . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION} && \
     nvm use v${NODE_VERSION} && \
     nvm alias default v${NODE_VERSION} && \
-    curl -o aws-ip-ranges.json https://ip-ranges.amazonaws.com/ip-ranges.json && \
-    bash /install_nginx.sh && \
+    curl -o aws-ip-ranges.json https://ip-ranges.amazonaws.com/ip-ranges.json
```
Comment (netsettler) [inline Dockerfile:51]: So you want to run the nginx install regardless of whether this curl succeeds?

File: deploy/docker/production/install_nginx.sh [hunk 3]
```diff
@@ -8,18 +8,17 @@ export PKG_RELEASE=1~buster
...
     found=''; \
     for server in \
-        ha.pool.sks-keyservers.net \
         hkp://keyserver.ubuntu.com:80 \
```
Comment (netsettler) [inline deploy/docker/production/install_nginx.sh:16]: weird that a security operation is on port 80. I assume there's not an https/443 option?

File: deploy/docker/production/install_nginx.sh [hunk 4]
```diff
@@ -8,18 +8,17 @@ export PKG_RELEASE=1~buster
...
-        hkp://p80.pool.sks-keyservers.net:80 \
-        pgp.mit.edu \
+        pgp.mit.edu:80 \
     ; do \
         echo "Fetching GPG key $NGINX_GPGKEY from $server"; \
-        apt-key adv --keyserver "$server" --keyserver-options timeout=10 --recv-keys "$NGINX_GPGKEY" && found=yes && break; \
+        gpg --keyserver "$server" --keyserver-options timeout=10 --recv-keys "$NGINX_GPGKEY" && found=yes && \\
```
Comment (netsettler) [inline deploy/docker/production/install_nginx.sh:20]: Do you really want two backslashes at the end of the line here?

**Review by netsettler — APPROVED** [review]
It looks like you're using a different procedure this time than last time. Without knowing the details of the protocol change I'm at a loss to verify the efficacy of the approach, but I assume you've tested that. I marked things that were syntactically questionable, but you can ignore anything that doesn't seem relevant. The \\ thing is the most likely to be a real problem. The rest is just me pointing to curiosities.

**Review by dmichaels-harvard — APPROVED** [review]

**Pull request merged**